app/src/main/java/com/codesoom/assignment/application/UserService.java

```diff
@@ -0,0 +1,86 @@
+package com.codesoom.assignment.application;
+
+import com.codesoom.assignment.UserNotFoundException;
+import com.codesoom.assignment.domain.User;
+import com.codesoom.assignment.domain.UserRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+
+/**
+ * 유저를 관리합니다.
+ */
+@Service
+public class UserService {
+
+    private UserRepository userRepository;
```

Comment:
일단 할당되면 누군가가 바꾸는 실수를 할 수 없도록 이렇게 `final`을 붙여줍시다.
```suggestion
    private final UserRepository userRepository;
```